src/alignof.cc

```diff
@@ -0,0 +1,68 @@
+
```

Comment:
No particularly good reason, but I was planning on using `process.binding('sizeof').pointer` within buffer.js in order to add correct offset validation to the read/writePointer() functions. We could easily just `require('os').sizeof.pointer` instead though. I'm open either way.